lib/app/features/feed/create_story/views/components/story_video/story_published_notification.dart

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:flutter_animate/flutter_animate.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:ion/app/effects/slide_in_out_effects.dart';
+import 'package:ion/app/extensions/extensions.dart';
+import 'package:ion/app/features/feed/create_story/providers/story_camera_provider.dart';
+import 'package:ion/generated/assets.gen.dart';
+
+class StoryPublishedNotification extends ConsumerWidget {
+  const StoryPublishedNotification({required this.height, super.key});
+
+  final double height;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final isStoryPublished = ref.watch(
+      storyCameraControllerProvider.select((state) => state.isStoryPublished),
+    );
+
+    return Animate(
+      effects: topSlideInOutEffects,
+      onComplete: (controller) =>
+          ref.read(storyCameraControllerProvider.notifier).resetStoryPublished(),
+      autoPlay: isStoryPublished,
+      child: SizedBox(
+        height: height,
```

Comment:
If the i18n text is long and might require multiple lines for different languages, we should ensure the layout is flexible enough to accommodate multiline text.

Reply:
yeah, we should test the app entirely to ensure the layouts are fine in all places

Reply:
@ice-orion  @ice-hades A fixed height is needed to move content below the notification on the Feed Page and avoid overlapping the notification with the content. Wrapping with a `Column` is not necessary in this case.